### ISSUE TITLE
Stairs on posts all face south

### ISSUE THREAD

**Issue by smellyonionman** [MLG-Fortress/PopulationDensity#63]
Looks like I'll be switching to slabs but I also wanted to let you know that in 1.13, all stairs on the outer or inner rings face the same way.  It looked a lot better in 1.12! Oh well.  Thanks for forking the plugin, it's still awesome.

**Comment by RoboMWM** [comment]
Yes, because you'd need to set blockdata, which I'm not sure if there's an easy way to do that programmatically besides inserting the raw string... and even then, it'd need to be set for each side.

Doesn't the post generate directly in the ground though? So stairs wouldn't make much sense in the first place?
